pkgs/tools/networking/networkmanager/default.nix

```diff
@@ -140,10 +146,15 @@ stdenv.mkDerivation rec {
     newt
     libsoup
     jansson
+    dbus
```

Comment:
Add a comment that this is only used for getting service installation directory paths from the pkg-config, since libdbus should not really be needed.